packages/acgtk/acgtk.2.0.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "sylvain.pogodalla@inria.fr"
+
+name: "acgtk"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+install: ["dune" "install"]
+
+depends: [
+  "ocaml" { >= "4.13.0" }
+  "dune" { >= "3.5" }
+  "menhir" { >= "20211230"}
+  "ocamlgraph"
+  "ANSITerminal" { >= "0.8" }
+  "fmt"
+  "logs"
+  "mtime" { >= "2.0.0"}
+  "cmdliner" { >= "1.1.0"}
+  "sedlex"
+  "cairo2"
+  "yojson" { >= "1.6.0"}
+  "readline"
+  "ocamlformat" { dev }
+  "odoc" { dev }
+]
+
+dev-repo: "git+https://gitlab.inria.fr/ACG/dev/ACGtk.git"
+
+homepage: "http://acg.loria.fr/"
+license: "CeCILL-1.0+"
+authors: ["Sylvain Pogodalla"]
+bug-reports: "sylvain.pogodalla@inria.fr"
+
+synopsis: "Abstract Categorial Grammar development toolkit"
+
+description: "This toolkit provides a compiler and an interpreter for Abstract Categorial Grammars (ACGs). Grammars can be compiled and then used by the interpreter to parse (if the grammar is at most second-order) or to generate terms. See http://acg.loria.fr for more details and bibliographic references."
+
+url {
+     src: "http://acg.loria.fr/software/acg-2.0.0-20231006.tar.gz"
+     checksum: "sha512=eb959a38e963cda523a694289edf7cd4f1103e48e7b2bff62d42ed19e4b8fa75ef17d3f8cbe8ac3dd1b8fd6cdae8938023682be89065c1213d135423b05cecff"
+}
```

Comment:
```suggestion
}
available: os != "freebsd"
```
as mentioned in https://github.com/ocaml/opam-repository/pull/24594#issuecomment-1751610413